landowner.h

```diff
@@ -0,0 +1,39 @@
+#include <iostream>
+#include <vector>
+
+class Landowner
+/**/
+{
+private:
+  std::string m_Name,m_Sex;
+  int m_Gold;
+  long m_Exp;
+  std::vector<int> packCards;//Default deck of cards(1-54);
+  std::vector<int> surplusCards;//Surplus of cards
+  std::vector<int> currCards;//Current player's hand
+  bool isContains(int); //If the surplus contains the card 
+  void deleteCard(std::vector<int>&,int);
+  std::string getColor(int);
```

Comment:
const the function everywhere, don't use `get` or `set` in the name, use operator overloaidng instead.